quickstarts/microsoft.web/web-app-asp-app-on-asev3-create/main.bicep

```diff
@@ -121,15 +110,12 @@ resource virtualnetwork 'Microsoft.Network/virtualNetworks@2020-11-01' = if (!us
       }
     }]
   }
-  dependsOn: [
```

Comment:
revert this change and move the ```dependsOn``` property above the ```properties``` property (i.e. put it back and change the order)

Reply:
done.